models/education-nationale.ts

```diff
@@ -0,0 +1,58 @@
+import { clientEducationNationale } from '#clients/education-nationale';
+import { HttpNotFound } from '#clients/exceptions';
+import { clientRGE } from '#clients/rge';
+import { EAdministration } from '#models/administrations';
+import {
+  APINotRespondingFactory,
+  IAPINotRespondingError,
+} from '#models/api-not-responding';
+import { Siren, verifySiren } from '#utils/helpers';
+import logErrorInSentry from '#utils/sentry';
+import { getUniteLegaleFromSlug } from './unite-legale';
+
+export interface IEducationNationaleEtablissement {
+  libelleAcademie?: string | null;
+  adresse?: string | null;
+  codePostal?: string | null;
+  nomCommune?: string | null;
+  mail?: string | null;
+  idEtablissement?: string | null;
+  nombreEleves?: number | null;
+  nomEtablissement?: string | null;
+  siret?: string | null;
+  statut?: string | null;
+  telephone?: string | null;
+  zone: string;
+}
+
+export const getUAI = async (
+  slug: Siren
```

Comment:
Type is already `Siren`

Prefer `async (siren: Siren) =>`

And skip the verifySiren as has already been verified on line48